src/index.js

```diff
@@ -30,6 +30,10 @@ setup(app)
 
 // Connect to Redis
 cache.setup(() => {
+  // if we can't get an xapp token, just exit and let the whole system try
+  // again - this prevents a sustained broken state when gravity returns a
+  // 502 during force startup.
+  artsyXapp.on('error', process.exit)
```

Comment:
Would it be possible to log some kind of helpful message if this happens? Just ran into a similar `.env` issue with @zephraph and it correctly exited but was a little hard to figure out why.

Reply:
yessssss... Hopefully that spits out an error message that we could console.log or something.

Reply:
I'll add this to https://github.com/artsy/force/pull/2829